src/main/resources/application.yaml

```diff
@@ -84,9 +84,22 @@ spring:
 idam:
   baseUrl: ${OPEN_ID_IDAM_URL:http://127.0.0.1:5000}
   redirectUrl: ${IA_IDAM_REDIRECT_URI:http://localhost:3002/oauth2/callback}
+  s2s-auth:
```

Comment:
@rajeshthuraiyur : Env variables all are being used same as ia-case-api. And if set up is same, we might end up using same values. If it is the intention, then all is fine. But in case, things diverge, should we have separate set of variables for bails-case-api?

Reply:
yes we should have separate variables for bails. We can consider raising a separate refactor ticket